### ISSUE TITLE
ERROR:composer.cli.launcher:Rank 0 crashed with exit code -11.

### ISSUE THREAD

**Issue by zhongxiaoyu** [mosaicml/diffusion#140]
@Skylion007 when i start train the sd-2 (vae.encoder step) ,the issue occurs, no any information but "ERROR:composer.cli.launcher:Rank 0 crashed with exit code -11." hope your reply,thanks.

Instantiating logger <composer.loggers.wandb_logger.WandBLogger>
[2024-05-07 09:39:45,349][composer.algorithms.low_precision_groupnorm.low_precision_groupnorm][INFO] - Successfully replaced 61 instances of GroupNorm with LowPrecisionGroupNorm
[2024-05-07 09:39:45,379][composer.algorithms.low_precision_layernorm.low_precision_layernorm][INFO] - Successfully replaced 48 instances of LayerNorm with LowPrecisionLayerNorm
Instantiating callbacks <composer.callbacks.speed_monitor.SpeedMonitor>
Instantiating callbacks <composer.callbacks.lr_monitor.LRMonitor>
Instantiating callbacks <composer.callbacks.memory_monitor.MemoryMonitor>
Instantiating callbacks <composer.callbacks.runtime_estimator.RuntimeEstimator>
Instantiating callbacks <composer.callbacks.OptimizerMonitor>
start to train........
[W Utils.hpp:135] Warning: Environment variable NCCL_ASYNC_ERROR_HANDLING is deprecated; use TORCH_NCCL_ASYNC_ERROR_HANDLING instead (function getCvarInt)
[2024-05-07 09:39:45,388][composer.utils.reproducibility][INFO] - Setting seed to 17
[2024-05-07 09:39:45,572][composer.trainer.trainer][INFO] - Run name: 1715045985-burrowing-goshawk
wandb: (1) Create a W&B account
wandb: (2) Use an existing W&B account
wandb: (3) Don't visualize my results
wandb: Enter your choice: 3
wandb: You chose "Don't visualize my results"
wandb: Tracking run with wandb version 0.16.3
wandb: W&B syncing is set to `offline` in this directory.
wandb: Run `wandb online` or set WANDB_MODE=online to enable cloud syncing.
wandb: WARNING URL not available in offline run
                                                                                                                                                      [2024-05-07 09:40:08,246][composer.trainer.trainer][INFO] - Stepping schedulers every batch. To step schedulers every epoch, set `step_schedulers_every_batch=False`.
[2024-05-07 09:40:08,249][composer.utils.reproducibility][INFO] - Setting seed to 17
/home/jinglianzhong/miniforge3/envs/py39/lib/python3.9/site-packages/torch/distributed/fsdp/_init_utils.py:437: UserWarning: FSDP is switching to use `NO_SHARD` instead of ShardingStrategy.SHARD_GRAD_OP since the world size is 1.
  warnings.warn(
[2024-05-07 09:40:08,308][composer.trainer.trainer][INFO] - Setting seed to 17
[2024-05-07 09:40:08,308][composer.utils.reproducibility][INFO] - Setting seed to 17
[2024-05-07 09:40:08,308][composer.trainer.trainer][INFO] - Using precision Precision.AMP_FP16
******************************
Config:
composer_commit_hash: None
composer_version: 0.22.0
node_name: unknown because NODENAME environment variable not set
num_gpus_per_node: 1
num_nodes: 1
rank_zero_seed: 17
time/remaining_estimate_unit: hours

******************************
                                                                                                                                                      self.image_key: image            0%|                         | 0/550000 [00:00<?, ?ba/s]
c: <torch.cuda.amp.autocast_mode.autocast object at 0x7f013c97c820>

ERROR:composer.cli.launcher:Rank 0 crashed with exit code -11.
Waiting up to 30 seconds for all training processes to terminate. Press Ctrl-C to exit immediately.
Global rank 0 (PID 1234168) exited with code -11
ERROR:composer.cli.launcher:Global rank 0 (PID 1234168) exited with code -11

**Comment by zhongxiaoyu** [comment]
wrong pytorch version
